app/Http/Transformers/ActionlogsTransformer.php

```diff
@@ -158,7 +158,6 @@ public function transformActionlog (Actionlog $actionlog, $settings = null)
                 [
                     'url' => $file_url,
                     'filename' => $actionlog->filename,
-                    'inlineable' => (bool) Helper::show_file_inline($actionlog->filename),
```

Comment:
Are we really, 100% sure, that this attribute isn't used - or attempted to be used - anywhere?

Reply:
I'm not, actually. I do see it in BS tables, but it's possible we weren't using it properly before. Easy enough fix if we were/

Reply:
It's actually a bit trickier than I thought, and I'm not sure we were using it properly anyway. I'll revisit in another PR.